### PR TITLE
Print quiz

### PR DESCRIPTION
**Pull request by BenHSmith13**
https://www.pivotaltracker.com/story/show/136803377

### CODE REVIEW

File: client/js/quiz_print/components/question_types/text_only_question.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React    from 'react';
+
+export default function essayQuestion(props) {
+  return (
+    <div>
```
Comment (BenHSmith13) [inline client/js/quiz_print/components/question_types/text_only_question.jsx]: kill outer div, make this file used in other files

File: client/js/quiz_print/reducers/print.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      return newState;
+    }
+
+    case 'LIST_QUESTIONS_IN_QUIZ_OR_SUBMISSION_DONE': {
+      const newState = _.cloneDeep(state);
+      _.forEach(action.payload, (question) => {
+        if (!newState.questions[question.quiz_id]) {
+          newState.questions[question.quiz_id] = {};
+        }
+        newState.questions[question.quiz_id][question.id] = question;
```
Comment (BenHSmith13) [inline client/js/quiz_print/reducers/print.js]: string these

**Comment by BenjaminCall** [comment]
looks purtty good.

**Comment by emdurr** [comment]
Looks good to me.

**Pull request merged**